pkg/cmd/alias/imports/import_test.go

```diff
@@ -315,6 +325,9 @@ func TestImportRun(t *testing.T) {
 			apiCmd := &cobra.Command{Use: "api"}
 			apiCmd.AddCommand(&cobra.Command{Use: "graphql"})
 			rootCmd.AddCommand(apiCmd)
+			for _, cmd := range tt.aliasCommands {
+				rootCmd.AddCommand(cmd)
+			}
```

Comment:
All the tests were previously passing because these alias commands were not set up properly.